integration-tests/hibernate-orm-panache/pom.xml

```diff
@@ -109,6 +119,30 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <executions>
+                    <!--
+                    The prod mode tests need to be part of a different execution to ensure that they don't mess with the standard tests.
+                    By adding this configuration we ensure that the maven surefire plugin will execute twice, one for the regular **/*Test.java
+                    tests (using the 'default-test' execution), and one for the prod mode tests (this 'prod-mode' execution)
+                    -->
```

Comment:
Are we sure we need to split the test runs in two parts?

Reply:
Yes, this is needed because of classpath issues - I've encountered it before.

Reply:
and in this case it wasn't for the classpath but because the regular tests already start an H2 database instance, and the *PMT test run on its own JVM, so there are TCP port conflicts.

Reply:
Right, good find